ui/src/components/display-text/display-text.tsx

```diff
@@ -0,0 +1,14 @@
+import { DisplayTextStyles as S } from './display-text.styles';
+type DisplayTextProps = {
+  label: string;
+  value: string | React.ReactNode;
+};
+
+export const DisplayText: React.FC<DisplayTextProps> = ({ label, value }) => {

```

Comment:
This is a not customizable component, maybe it should be a fragment in the view folder.